doc/latex.rst

```diff
@@ -1065,41 +1015,43 @@ Do not use quotes to enclose values, whether numerical or strings.
        ``div.note_TeXextras`` (etc) described in :ref:`additionalcss`.
 
 ``noteborder``, ``hintborder``, ``importantborder``, ``tipborder``
-    The width of the two horizontal rules.
-
-    If the background color is set, or the alternative :ref:`additionalcss`
-    syntax is used (e.g. ``div.note_border-width=1pt`` in place of
-    ``noteborder=1pt``), or *any* option with a CSS-alike name is used, then
-    the border is a full frame and this parameter sets its width also for left
-    and right.
+    The width of the border.
```

Comment:
We should add: "the four sides can be configured separately using CSS-alike options described later" (with a link).